v0/destinations/active_campaign/transform.js

```diff
@@ -116,7 +116,12 @@ const customTagProcessor = async (message, category, destination) => {
         }
       }
     );
-  } catch (err) {}
```

Comment:
Modify the test suite for all the cases we updated the code for.

Reply:
adding failing test case and mocks response in transformer for network call which we make to the active campaign can be challenging as it can fail due to a number of reasons one being network failure.

Reply:
I see.